cves/kernel/CVE-2013-3223.yml

```diff
@@ -186,8 +221,11 @@ specification:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  note:
-  answer:
+  answer: false
+  note: |
+    No violation of specifications, the socket driver simply neglected to
```

Comment:
I would also suggest including where you have looked for any violation of specifications in commit messages or bug reports and then state this statement.